# mftb already refers to time base lower, mftbl is invalid

--- tests/perf_private.h.orig	2022-02-02 04:16:44.000000000 +0800
+++ tests/perf_private.h	2022-05-29 08:01:07.000000000 +0800
@@ -101,7 +101,7 @@
     __asm__ volatile (\
       "0:\n"
       "mftbu %0\n"
-      "mftbl %1\n"
+      "mftb %1\n"
       "mftbu %2\n"
       "cmpw %0, %2\n"
       "bne- 0b"
